common/src/main/java/whocraft/tardis_refined/common/entity/ControlEntity.java

```diff
@@ -152,10 +169,10 @@ public InteractionResult interactAt(Player player, Vec3 vec3, InteractionHand in
 
                 if (player.getMainHandItem().getItem() == Items.DEBUG_STICK) {
                     if (player.getOffhandItem().getItem() == Items.REDSTONE) {
-                        float x = (float) ( this.position().x - 0.5f);
-                        float y = (float) ( this.position().y - 97.5f);
-                        float z = (float) ( this.position().z - -4.5f);
-                        System.out.println("Output: " + x +"f, "+ y +"f, "+ z +"f");
+                        float x = (float) (this.position().x - 0.5f);
+                        float y = (float) (this.position().y - 97.5f);
+                        float z = (float) (this.position().z - -4.5f);
+                        System.out.println("Output: " + x + "f, " + y + "f, " + z + "f");
```

Comment:
Remove prints

Reply:
Swapped to debug logging as this is something we want to keep.